docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -82,7 +82,9 @@ property key | Description | Default |
 `collector.backend_service`|Collector SkyWalking trace receiver service addresses.|`127.0.0.1:11800`|
 `logging.level`|The log level. Default is debug.|`DEBUG`|
 `logging.file_name`|Log file name.|`skywalking-api.log`|
+`logging.output`| Log output. Default is FILE. Use CONSOLE means output to stdout. |`FILE`|
 `logging.dir`|Log files directory. Default is blank string, means, use "system.out" to output logs.|`""`|
+`logging.pattern `|logging format. Default is blank string, means, use EasyLogger who pattern is `%level %timestamp %thread %class : %msg %throwable` |`""`|
```

Comment:
Add document about pattern options. Are these(%level %timestamp %thread %class : %msg %throwable) all options? If so, make it clear in the document.